core/includes/config.inc

```diff
@@ -145,6 +155,11 @@ function config_get_translated($config_file, $option = NULL, $args = array(), $o
  *   may contain periods to indicate levels within the config file.
  * @param mixed $value
  *   The value to save into the config file.
+ *
+ * @see config_get()
+ * @see config_diff()
+ * @see config_clear()
+ * @see config_menu()
```

Comment:
And again here, I don't think `config_diff()` and `config_menu()` are necessary here.

Reply:
Agree